docs/resources.md

```diff
@@ -290,171 +271,6 @@ You can refer to different examples demonstrating usage of optional resources in
 
 ## Resource Types
 
-### Git Resource
```

Comment:
There are a few other places in this doc where the instructions refer to `git PipelineResource`. Its not a blocker in merging this PR but at this point, `resources.md` does not bring much value. If we are not too far from deleting `resources.md`, its reasonable to make these changes.